app/javascript/src/components/home-title.vue

```diff
@@ -0,0 +1,97 @@
+<template>  
+  <div class="home-title">
+    <div class="home-title__title">
+      Busquemos un regalo para
+      <span class="home-title__user-name">{{ receiverName }}</span>
+      <img
+          class="home-title_icon home-title__icon--clear"
+          src="../assets/close-badge.svg"
```

Comment:
Expected indentation of 8 spaces but found 10 spaces                       vue/html-indent